dhcpv6/types.go

```diff
@@ -5,6 +5,8 @@ package dhcpv6
 type MessageType uint8
 
 const (
+	// MSGTYPE_NONE is used internally and is not part of the RFC
+	MSGTYPE_NONE        MessageType = 0
```

Comment:
Should these be something like `MsgTypeNone` per go const style? Probably doesn't belong in this PR but a follow-up.

Reply:
yes, it's tracked in https://github.com/insomniacslk/dhcp/issues/34